examples/synthea-breast-cancer/Ali_Laiba_syntheaEDA/SyntheaClassifiers.ipynb

```diff
@@ -0,0 +1,1710 @@
+    "- [Accuracy](https://en.wikipedia.org/wiki/Accuracy_and_precision): Proportion of results the classifier correctly identifies\n",
+    "- [Macro Average](https://datascience.stackexchange.com/questions/65839/macro-average-and-weighted-average-meaning-in-classification-report): Unweighted Average F1 Score of all the classes\n",
+    "- [Weighted Average](https://datascience.stackexchange.com/questions/65839/macro-average-and-weighted-average-meaning-in-classification-report): Weighted Average F1 Score of all the classes\n",
+    "- [Balanced Accuracy](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.balanced_accuracy_score.html#sklearn.metrics.balanced_accuracy_score): Average of recall on both classes\n",
+    "- [Matthews Correlation Coefficient (MCC)](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.matthews_corrcoef.html): A balanced classification metric ranging from -1 (inverse prediction), 0 (average random prediction) to +1 (perfect prediction). This metric can be used to evaluate a model, even when the class sizes vary quite a bit in size, like ours do\n",
+    "- [Cross Validation](https://scikit-learn.org/stable/modules/cross_validation.html) A solution to overfitting within models. The resulting performance metric gives a better picture of the accuracy of the model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def print_results(y_test, y_pred):\n",
+    "    print('CONFUSION MATRIX:')\n",
+    "    print(confusion_matrix(y_test, y_pred))\n",
+    "    print()\n",
+    "    \n",
+    "    print('CLASSIFICATION REPORT:')\n",
+    "    print(classification_report(y_test, y_pred, zero_division=1, target_names=['Stage 1', 'Stage 2', 'Stage 3', 'Stage 4']))\n",
+    "    print()\n",
+    "    \n",
+    "    results_dict = {}\n",
+    "    results_dict['Balanced Accuracy'] = balanced_accuracy_score(y_test, y_pred)\n",
+    "    results_dict['Macro F1 Score'] = f1_score(y_test, y_pred, average='macro')\n",
+    "    results_dict['MCC'] = matthews_corrcoef(y_test, y_pred)\n",
+    "    results_df = pd.DataFrame([results_dict])\n",
+    "    \n",
+    "    print(results_df.to_string(index=False))\n",
+    "\n",
+    "def print_cross_validation(model, X, y):\n",
+    "    print()\n",
+    "    print(f'AVERAGE CROSS-VALIDATION SCORE: {cross_val_score(model, X, y).mean()}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Building our Models**\n",
+    "We will now go through our process of building several different models to determine the best fit for our data.\n",
+    "\n",
+    "The order in which we will build the models is as follows:\n",
```

Comment:
Thank you so much for organizing this section with a table of contents. Very neat.